app/src/typo.scss

```diff
@@ -16,3 +17,35 @@ $itero-mat-typography: mat-typography-config(
   $button:    mat-typography-level($fontsize, 1.2),
   $input:     mat-typography-level($fontsize, 1.3),
 );
+
+:root{
+  --font-size: 13pt;
+  --small-font-size: 12pt;
+}
```

Comment:
These values are never used (except on non-screen display), because for all screen widths, there is a matching @media below that overwrite the values.